Makefile

```diff
@@ -372,7 +372,7 @@ test-ci: | clear-stalled build-addons build-addons-napi
 	out/Release/cctest --gtest_output=tap:cctest.tap
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) -p tap --logfile test.tap \
 		--mode=release --flaky-tests=$(FLAKY_TESTS) \
-		$(TEST_CI_ARGS) $(CI_ASYNC_HOOKS) $(CI_JS_SUITES) known_issues
+		$(TEST_CI_ARGS) $(CI_ASYNC_HOOKS) $(CI_JS_SUITES) $(CI_NATIVE_SUITES) known_issues
```

Comment:
something I just noticed, could you switch:

`CI_NATIVE_SUITES := addons addons-napi`

to

`CI_NATIVE_SUITES ?= addons addons-napi`

while you're at it; that would allow this set of tests to be more easily turned off when running coverage locally. tldr; I think this file got slightly screwed up during rebase.

Reply:
Done